<?xml version="1.0"?>
<launch>

  <!-- MW-AHRS Driver -->
  <node pkg="mw_ahrs_ros" type="mw_ahrs_driver_node" name="mw_ahrs_driver_node" output="screen">
    <param name="port" value="/dev/ttyUSB0" />
    <param name="frame_id" value="imu_link" />
    <param name="version" value="v2" /> <!-- v1, v2 -->
  </node>

</launch>